administrator-manual/en/subscription.rst

```diff
@@ -18,7 +18,11 @@ getting access to monitoring portal and stable update repositories.
 Activating a subscription will enable the stable YUM repositories, but will
 disable any other repositories you may have added. You can re-enable any other
 repositories by creating a "template-custom" for
-:file:`/etc/nethserver/eorepo.conf`.
+:file:`/etc/nethserver/eorepo.conf` or by the command line (beware of case sensitive) 
```

Comment:
I'd remove the template-custom part and refactor to something like:
```
You can re-enable any other
repositories by setting ``ExtraRepository`` property from command line.
The property accept a comma-separated list of repository names and it's case sensitive. Example: ::
```

Reply:
yep I thought on it, no problem I will do, mainly I stated to let it because it is still true :D